syncstorage/storage/memcached.py

```diff
@@ -880,6 +931,85 @@ def del_item(self, userid, item):
             raise ItemNotFoundError
         return modified
 
+    def get_batches_key(self, userid):
+        return _key(userid, "c", self.collection, "batches")
+
+    def get_cached_batches(self, userid):
+        return self.cache.gets(self.get_batches_key(userid))
+
+    def create_batch(self, userid):
+        bdata, bcasid = self.get_cached_batches(userid)
+        batch = get_timestamp()
+        batchid = int(batch * 1000)
+        if not bdata:
+            bdata = {}
+        if batchid in bdata:
+            raise ConflictError
+        bdata[batchid] = {"user": userid,
+                          "modified": batch,
```

Comment:
AFAICT we do not use either of the "user" or "modified" keys from this object.  If true we should just drop them and save the overhead.